theme_kea/static/src/scss/primary_variables.scss

```diff
@@ -109,21 +52,27 @@ $o-theme-font-configs: (
 
 $o-website-values-palettes: (
     (
-        'color-palettes-name': 'kea-1',
+        'color-palettes-name': 'bewise-2',
 
-        'header-font-size': 1.25rem,
         'logo-height': 3rem,
         'fixed-logo-height': 3rem,
+        // Fonts
+        'header-font-size': 1rem,
         'font-size-base': .925rem,
         'font': 'Roboto',
-        'headings-font': 'Barlow Condensed',
-        'navbar-font': 'Barlow Condensed',
-        'buttons-font': 'Barlow Condensed',
+        'headings-font': 'Poppins',
+        'navbar-font': 'Poppins',
+        'buttons-font': 'Poppins',
+        // Links / Buttons
         'link-underline': 'never',
-        'header-links-style': 'outline',
-        'btn-primary-outline': true,
-        'header-template': 'default',
-        'footer-template': 'call_to_action',
+        'btn-border-radius': 10rem,
+        // Header
+        'header-links-style': 'pills',
+        'header-template': 'Contact',
```

Comment:
Same here, you set header-template to `'Contact'` but I don't see you enabling the contact header view, what am I missing?